src/components/SearchResults/SearchResults.js

```diff
@@ -222,6 +224,12 @@ const StyledTag = styled(Tag)`
 const ResultTag = styled('div')`
   display: flex;
   flex-direction: row;
+  padding-top: 21px;
```

Comment:
```suggestion
  padding-top: ${theme.size.default};
  align-items: center;
```

seeing the tags a little off center:

![image](https://github.com/mongodb/snooty/assets/13054820/2b7fca5e-bd4c-4b7d-a46d-3b6004a5b485)

Reply:
we should remove the lines 216-218 since ResultTag has the padding already